packages/server/test/unit/browsers/browsers_spec.js

```diff
@@ -30,6 +30,8 @@ after(() => {
   process.versions.electron = originalElectronVersion
 })
 
+const ctx = { browser: { setBrowserStatus () {} } }
```

Comment:
Might be worth standardizing on this function to make contexts for testing: https://github.com/cypress-io/cypress/blob/10.0-release/packages/data-context/test/unit/helper.ts#L24-L41

Should be as easy as importing and using it!

Reply:
Done in [3256e0f](https://github.com/cypress-io/cypress/pull/20563/commits/3256e0f3565f2c04d01114bd512563748fb6c953)